### PR TITLE
issue-51: Resolved IE10 table insertion issue

### PR DESCRIPTION
**Pull request by vikasrj**
@j0k3r Please find pull request for #51

### CODE REVIEW

**Comment by j0k3r** [comment]
Please fix Travis first.

File: src/js/grid.js [hunk 1]
```diff
@@ -18,17 +18,25 @@ Grid.prototype = {
...
+                    column: parseInt(el.getAttribute('data-column'), 10),
+                    row: parseInt(el.getAttribute('data-row'), 10)
                 },
                 active = this._currentCell &&
                          cell.row <= this._currentCell.row &&
                          cell.column <= this._currentCell.column;
 
             if (active === true) {
-                el.classList.add('active');
+                if ('classList' in document.createElement('div')) {
```
Comment (j0k3r) [inline src/js/grid.js]: Can't you do `if ('classList' in el) {` instead?

Comment (vikasrj) [reply src/js/grid.js]: sure, will do that change.

File: src/js/grid.js [hunk 2]
```diff
@@ -92,15 +100,15 @@ Grid.prototype = {
         var self = this,
             timer;
 
-        el.addEventListener('mouseenter', function () {
+        el.addEventListener('mouseenter', function (e) {
             clearTimeout(timer);
 
             var dataset = this.dataset;
```
Comment (j0k3r) [inline src/js/grid.js]: Since you don't use dataset, I think you can remove that line

**Review by j0k3r — APPROVED** [review]
LGTM

**Pull request merged**